src/index.js

```diff
@@ -39,12 +40,16 @@ class List extends React.Component {
 		})
 	}
 
-	render() {
+  updateList(event) {
+
+  }
 
+	render() {
 		let list = this.state.list;
 		let listItem = list.map((li, id) =>
 		  <li key={id} className="list-group-item d-flex justify-content-between align-items-center">
 		    <Link value={li} />
+        <button onClick={this.updateList} value={li} className="badge badge-primary badge-pill">Xóa</button>
```

Comment:
có cách viết khác không mà không dùng this không ?

Reply:
```suggestion
        <button onClick={(e) => this.updateList(e)} value={li} className="badge badge-primary badge-pill">Xóa</button>
```
hoặc
updateList = () => {
 // code
}

Reply:
viết vậy nghĩa là sao,